infer.py

```diff
@@ -10,6 +10,7 @@
 import astor
 import torch
 import tqdm
+from multiprocessing import Pool, Manager, Process
```

Comment:
```suggestion
import multiprocessing
```
In the rest of the project, all other imports are of modules.

Reply:
Oh ok, fixed